reconcile/utils/terraform_client.py

```diff
@@ -203,6 +213,9 @@ def log_plan_diff(self, name, tf, enable_deletion):
                 with self._log_lock:
                     logging.info([action, name, resource_type, resource_name])
                     self.should_apply = True
+                if action == 'create':
+                    if resource_type == 'aws_iam_user_login_profile':
+                        created_users.append(AccountUser(name, resource_name))
```

Comment:
This is an `and`, which is easier to read.
```suggestion
                if action == 'create' and resource_type == 'aws_iam_user_login_profile':
                    created_users.append(AccountUser(name, resource_name))
```

Reply:
perhaps, but more cases will probably be added over time (look at other sections above and below)